### DISCUSSION TITLE
Internationalize issue title and comments

### DISCUSSION THREAD

**Discussion by developersHub2b** [upptime/upptime#980] [Q&A] [answered]
I'd like to customize the title and description of the auto-generated issues e.g. "Placeholder is down" or "Resolved: Placeholder is back up in [134c769] after 32 minutes." but couldn't find anything on the docs, is that currently possible?

**Comment by AnandChowdhary** [comment, marked as answer]
You're right, the docs are not great for this, sorry about that!

Here are the internationalization docs that will tell you how to change the strings we use: https://upptime.js.org/docs/configuration/#internationalization

You can use the i18n object `commitMessages` like:

```yaml
commitMessages:
  statusChange: "$PREFIX $SITE_NAME is $STATUS ($RESPONSE_CODE in $RESPONSE_TIME ms) [skip ci] [upptime]"
```

(this is the default value of the commit message) and you can change this to change the commit title (the source code is https://github.com/upptime/uptime-monitor/blob/2735e1b2bb69e63b987844dd8db29a0315e66b33/src/update.ts#L338). Unfortunately, there is no way to change the issue title and description, it is hardcoded here (https://github.com/upptime/uptime-monitor/blob/2735e1b2bb69e63b987844dd8db29a0315e66b33/src/update.ts#L396).

**Reply by AnandChowdhary** [reply]
I think it's a great idea to add full support for i18n for the issue title and description. Please feel free to create a PR in the repository, changing the lines I pointed to here. :)
